.scp/settings.yaml

```
@@ -0,0 +1,209 @@
+# Smart Code Prompts - Project Settings
+# This file configures the behavior of the application for this specific project.
+# You can edit prompts and settings here. Changes are applied when you save the file.
+
+# SECURITY: API Key Configuration
+# The OpenRouter.ai API key should be loaded securely at runtime from environment variables.
+# DO NOT place API keys directly in this file or any version-controlled files.
+# 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)